### PR TITLE
[#1884] Provide instructions for SQLite but no hope

### PR DESCRIPTION
**Pull request by fwolfst**
# Release Notes

Added a file under doc/ with instructions on how to use nucore with a sqlite-database. For some developers (me ...) this might be the prefered way over using a MySQL or Oracle setup. However it has to be made clear that this is necessarily a good idea for production use.

To be fair I should note that I did not run tests and used ruby 2.5.1 .

# Screenshot

Optional.

# Additional Context

Optional. Feel free to add/modify additional headers as appropriate, e.g. "Refactorings", "Concerns".

### CODE REVIEW

**Comment by vandrijevik** [comment]
Hi @fwolfst,

Thank you very much for your support of `nucore-open` and this contribution!

Unfortunately, I don’t think it is best to merge this documentation, since we do not support SQLite in any way, and we should not be implying that we do (which we fear we would be even if there are clear warnings in the documentation).

We are currently experimenting with using Docker in some capacity for this project, which would make installing MySQL even easier than it is today, so we recommend that you use MySQL for development to avoid spending time tracking down bugs or differences that may arise if you use SQLite.

Once again, thank you for your work and this contribution!

Best,
Vlad

**Comment by fwolfst** [comment]
Understandable. I personally prefer quick Docker-less, SQLite-setups for hacking and evaluating, but thank god that tastes differ.

I do not agree with your risk assessment, but also I don't have to deal with support and issues myself :)
Btw, if there would have been a demo-instance running somewhere, I wouldnt have had the wish to trick around.

TL;DR: All good, I understand if you dont want to merge.

**Comment by vandrijevik** [comment]
Thanks for your understanding, @fwolfst, and I certainly agree that different tastes are a good thing 😄
